lib/Source/io/newgrounds/utils/AsyncHttp.hx

```diff
@@ -5,7 +5,9 @@ import io.newgrounds.NGLite;
 import haxe.Http;
 import haxe.Timer;
 
-#if neko
+#if (target.threaded)
```

Comment:
Had no idea you could do that

Reply:
https://haxe.org/manual/std-threading.html